### PR TITLE
Add Search, Loader and Toasts

### PR DESCRIPTION
**Pull request by AnkurChunekar**
- Added Search Functionality.
- Added a Simple Loading Spinner.
- Added Toast Component.

### CODE REVIEW

File: src/components/card/services/addToCart.service.js [hunk 1]
```diff
@@ -12,11 +13,13 @@ const addToCartService = async ({ url, body, token, successStatus, cartDispatch,
...
             });
             setCtaBtnText("Go To Cart");
+            toast.success(body.product.title.slice(0, 13).trim() + "... Added to Cart");
         } else {
             throw new Error(response.status, "<-- error code");
         }
     } catch (error) {
-        alert(error);
+        toast.error("Error Occured! Please Try Again.");
+        console.error(error)
```
Comment (AtulPant2704) [inline src/components/card/services/addToCart.service.js]: You can get error from the backend itself just write
`toast.error(error.response.data.errors[0])`

**Review by AtulPant2704 — APPROVED** [review]
All the functionalities are working fine, on error show msg from the backend itself, rest all looks good.

**Review by VishalPatil18 — APPROVED** [review]
The code looks quite good and works well. Great work @AnkurChunekar 👍

File: src/components/card/services/addToCart.service.js (hunk 1 above)
Comment (Jay-Gadhiya) [inline src/components/card/services/addToCart.service.js:22]: you can remove consoles.

Comment (AnkurChunekar) [reply src/components/card/services/addToCart.service.js]: Thanks for the suggestion.

File: src/components/loaders/circular/CircularLoader.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
+.lds-roller {
```
Comment (Jay-Gadhiya) [inline src/components/loaders/circular/CircularLoader.css]: suggestion : you can give name, "loaders-roller"

**Review by Jay-Gadhiya — APPROVED** [review]

File: src/components/card/services/addToCart.service.js (hunk 1 above)
Comment (AnkurChunekar) [reply src/components/card/services/addToCart.service.js:22]: This here is not a `console.log`, it's `console.error()` method which is used to display error in  the console.

**Pull request merged**